code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -156,6 +160,7 @@
 				/obj/item/clothing/gloves/color/random = 8,
 				/obj/item/clothing/shoes/laceup = 1,
 				/obj/item/weapon/storage/secure/briefcase = 3,
+				/obj/item/weapon/relic = 3,
```

Comment:
and relics??i actually think we should leave relics out,tg nullrodbased relics are better